test/shared/commonFunctionsForTests.sh

```diff
@@ -45,9 +50,12 @@ CMP_FAIL_STR='Configure & Build.....FAIL'
 EXE_PASS_STR='Execute Simulation....PASS'
 EXE_FAIL_STR='Execute Simulation....FAIL'
 EXE_TBD_STR='Execute Simulation....TBD'
-EXE_BUILD_LIST=("default" "apm"   "carbon"  "hg"       \
-                "luowd"   "rrtmg" "tomas15" "tomas40" )
-
+EXE_GCC_BUILD_LIST=("default" "apm"   "carbon"  "hg"       \
+                    "luowd"   "rrtmg" "tomas15" "tomas40" )
+EXE_GCHP_BUILD_LIST=("default" "rrtmg")
```

Comment:
Should we add carbon here to test that it at least compiles for GCHP?

Reply:
This is now updated in commit da258592

Reply:
Added in commit da25859232112f5bad1c2e3ad235db0c71048620